BlackJack/index.js

```diff
@@ -29,189 +41,295 @@ makeCard();
 
 //onpage card
 let dealerRandomCard = Math.floor(Math.random() * allCard.length);
-dealerImg.src = `cards/${allCard[dealerRandomCard]}`
-dealerCards.appendChild(dealerImg)
-let dealerCardN = allCard[dealerRandomCard]
-let dealerCardNum = dealerCardN.charAt(0)
+dealerImg.src = `cards/${allCard[dealerRandomCard]}`;
+dealerCards.appendChild(dealerImg);
+let dealerCardN = allCard[dealerRandomCard];
+let dealerCardNum = dealerCardN.charAt(0);
 
 //dealer hidden card
 // let dealerHiddenRandomCard = Math.floor(Math.random() * allCard.length) +1;
 
-
 let playerRandomCard = Math.floor(Math.random() * allCard.length);
-playerImg.src = `cards/${allCard[playerRandomCard]}`
-playerCards.appendChild(playerImg)
-
-let playerRandomCard2 = Math.floor(Math.random() * allCard.length) +1;
-playerImg2.src = `cards/${allCard[playerRandomCard2]}`
-playerCards.appendChild(playerImg2)
+playerImg.src = `cards/${allCard[playerRandomCard]}`;
+playerCards.appendChild(playerImg);
 
+let playerRandomCard2 = Math.floor(Math.random() * allCard.length) + 1;
+playerImg2.src = `cards/${allCard[playerRandomCard2]}`;
+playerCards.appendChild(playerImg2);
 
 //save player first card value
-let firstCardN = allCard[playerRandomCard]
-let secondCardN = allCard[playerRandomCard2]
-let firstCardNum = firstCardN.charAt(0)
-let secondCardNum = secondCardN.charAt(0)
+let firstCardN = allCard[playerRandomCard];
```

Comment:
Variable naming could be better. E.g.: playerRandomCard here is actually the index. Hence something like playerRandomCardIndex would be clearer, and instead of firstCardN, it will then be firstCard to store the card. As for DOM variable, I always use something like firstCardElement, so its clear if its an index, or element or anything else